core/src/main/java/com/linecorp/armeria/client/Http1ResponseDecoder.java

```diff
@@ -211,8 +211,13 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
                         if (dataLength > 0) {
                             assert res != null;
                             final long maxContentLength = res.maxContentLength();
-                            if (maxContentLength > 0 && res.writtenBytes() > maxContentLength - dataLength) {
-                                fail(ctx, ContentTooLargeException.get());
+                            final long transferredLength = res.writtenBytes();
+                            if (maxContentLength > 0 && transferredLength > maxContentLength - dataLength) {
+                                fail(ctx, ContentTooLargeException.builder()
+                                                                  .maximum(maxContentLength)
+                                                                  .transferring(dataLength)
```

Comment:
How about `limit` and `delta`?
```suggestion
                                                                  .limit(maxContentLength)
                                                                  .delta(dataLength)
```

Reply:
Renamed